tests/data.py

```diff
@@ -2,14 +2,21 @@
 
 
 class Actor(Enum):
-    # User ID, org code, business function, OBO User ID
-    RC = ("555031999105", "D82106", "REFERRING_CLINICIAN")
-    RCA = ("555031998104", "D82106", "REFERRING_CLINICIAN_ADMIN")
-    RA = ("555031998103", "D82106", "REFERRING_ADMIN")
-    SPC = ("555032006106", "RCD", "SERVICE_PROVIDER_CLINICIAN")
-    SPA = ("555032006103", "RCD", "SERVICE_PROVIDER_ADMIN")
-    SPCA = ("555031998104", "RCD", "SERVICE_PROVIDER_CLINICIAN_ADMIN", "555031999105")
-    RC_DEV = ("021600556514", "R69", "REFERRING_CLINICIAN")
+    # User ID, org code, business function, IAL, OBO User ID
+    RC = ("555031999105", "D82106", "REFERRING_CLINICIAN", "2")
+    RCA = ("555031998104", "D82106", "REFERRING_CLINICIAN_ADMIN", "3")
+    RA = ("555031998103", "D82106", "REFERRING_ADMIN", "3")
+    SPC = ("555032006106", "RCD", "SERVICE_PROVIDER_CLINICIAN", "3")
+    SPA = ("555032006103", "RCD", "SERVICE_PROVIDER_ADMIN", "3")
+    SPCA = (
+        "555031998104",
+        "RCD",
+        "SERVICE_PROVIDER_CLINICIAN_ADMIN",
+        "3",
+        "555031999105",
+    )
+    RC_DEV = ("021600556514", "R69", "REFERRING_CLINICIAN", "3")
+    RC_IAL2 = ("555031999105", "D82106", "REFERRING_CLINICIAN", "2")
 
```

Comment:
should 'RC' and RC_IAL2 have the same IAL ?
If they do - do we need them both ?

Reply:
Good point. RC_IAL2 removed.